.github/actions/create-jira-issues-for-community-activities/Initialize-IssueDetails.ps1

```diff
@@ -54,7 +54,7 @@ switch ($GitHub.event_name)
     default
     {
         $message = "Unknown event `"$($GitHub.event_name)`". Please only call this script for one of the following " +
-            "events: discussion, issues, pull_request."
+        "events: discussion, issues, pull_request."
```

Comment:
The original indentation was good, it's a continuation of the previous string.

Reply:
I agree. VS Code autoformatting changed this, but I don't know why. If you put brackets around it, then the indentation is correct, but I'll just revert this for now.